include/swift/Sema/Constraint.h

```diff
@@ -780,6 +788,10 @@ class Constraint final : public llvm::ilist_node<Constraint>,
   /// e.g. coercion constraint "as X" which forms a disjunction.
   bool isExplicitConversion() const;
 
+  /// Determine whether this constraint should be solved in isolation
+  /// from the rest of the constraint system.
```

Comment:
One-way constraints form a separate component for left-hand side type variable, in contrast isolated closures have to remove all of the inactive constraints and unrelated type variables and preserve only type variables related to parameter and result, they can't be separated into a component like one-way are because all of the constraints around a closure would have to be brought back after the body is type-checked.